### DISCUSSION TITLE
Please add video/background wallpaper to be pulled from computer files

### DISCUSSION THREAD

**Discussion by Lars-ellery02** [zombieFox/nightTab#343] [Ideas]
The extension is great and I love the amount of customization it has and Ive had no problems with it, but having the extension pull from files on your computer instead of the internet would be a lot easier (i tried to follow the tutorial, but i couldn't get links to work) Would you be doing something like this in the future?

**Comment by zombieFox** [comment]
Glad you like the project. This was possible in the past but the feature had to be dropped. The rational behind having to host images and videos is due to browser local storage limits (and a few other issues). The [background image wiki explains more](https://github.com/zombieFox/nightTab/wiki/Local-background-image).
